package.json

```diff
@@ -5,7 +5,7 @@
   "eyeglass": {
     "exports": "eyeglass-exports.js",
     "name": "ibm-colors",
-    "needs": "^0.8.3"
+    "needs": "^1.2.1"
```

Comment:
I think we probably just want this to be `^1.0.0`, to avoid people having to use too specific a version of eyeglass (i.e. they have to have the very latest in their dependencies at the moment).